common/constant.go

```diff
@@ -0,0 +1,5 @@
+package common
+
+const (
+	DefaultGasPriceMultiplier = 2 // the default gas price multiplier for outbond txs
```

Comment:
```suggestion
	// DefaultGasPriceMultiplier is the default gas price multiplier for outbond txs
	DefaultGasPriceMultiplier = 2
```
This is the standard way in Go

Reply:
[commit](https://github.com/zeta-chain/node/pull/1747/commits/2124f357744661b6646c324d2cb090cb27f844c1)